### PR TITLE
FedEx tracking response city and state/province defaults to unknown if empty

### PR DESCRIPTION
**Pull request by ZachGreen-ShipOffers**
if city or state/province in a fedex tracking response is blank defaults to unknown.

### CODE REVIEW

File: lib/active_shipping/carriers/fedex.rb [hunk 1]
```diff
@@ -539,23 +539,28 @@ def parse_transit_times(times)
...
+          args[:country] = node.at('CountryCode').text
+        else
+          args[:country] = ActiveUtils::Country.new(:alpha2 => 'ZZ', :name => 'Unknown or Invalid Territory', :alpha3 => 'ZZZ', :numeric => '999')
+        end
+        if node.at('StateOrProvinceCode')
+          args[:province] = node.at('StateOrProvinceCode').text
+        else
+          args[:province] = 'unknown'
+        end
+        if node.at('City')
```
Comment (eboswort) [inline lib/active_shipping/carriers/fedex.rb]: How would you feel about just:
`args[:city] = node.at('City').try(:text) || 'unknown'`

Comment (eboswort) [inline lib/active_shipping/carriers/fedex.rb]: That style could shorten this method quite a bit

**Comment by ZachGreen-ShipOffers** [comment]
added a minor change to allow tracking of mail innovations with ups.

**Comment by kmcphillips** [comment]
Closes #263

**Comment by kmcphillips** [comment]
Looks good :+1:

**Comment by ZachGreen-ShipOffers** [comment]
Thank you. Glad I could help

**Comment by kmcphillips** [comment]
Can you bump version to `1.1.3` and then I'll merge and release for you.

https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/version.rb

**Pull request merged**